### PR TITLE
 add : get_score(lat, lon)

### PR DESCRIPTION
**Pull request by GHCamille**
Add function to get the fwi score for a specific lat lon.

### CODE REVIEW

File: pyro_risks/platform_fwi/get_fwi_effis_score.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from datetime import date
+
+# Geographic librairies Imports
+import geopandas as gpd
+from shapely.geometry import Point, Polygon
+
+# Pyro Risks Imports
+from pyro_risks.utils.s3 import S3Bucket
+
+
```
Comment (Acruve15) [inline pyro_risks/platform_fwi/get_fwi_effis_score.py:13]: [MEDIUM] a good practice is to state function that should be used from the script. 

```suggestion
__all__ = ["get_score"]
```

File: pyro_risks/platform_fwi/get_fwi_effis_score.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Usual Imports
```
Comment (Acruve15) [inline pyro_risks/platform_fwi/get_fwi_effis_score.py:1]: [LOW] IMO these comments are not needed when importing libraries (maybe use `isort` to automatically sort your imports.

File: pyro_risks/platform_fwi/get_fwi_effis_score.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    year, month, day = retrieved_date.split("-")
+
+    json_content = s3.read_json_from_s3(
+        object_key=f"fwi/year={year}/month={month}/day={day}/fwi_values.json",
+    )
+
+    gdf = gpd.GeoDataFrame.from_features(json_content["features"])
+
+    gdf["fwi_category_for_point"] = gdf.apply(
+        lambda row: point_fwi_category(row, point_coords), axis=1
```
Comment (Acruve15) [inline pyro_risks/platform_fwi/get_fwi_effis_score.py:44]: [HIGH] `gdf["geometry"].contains(point)` does not work? Apply methods can be really slow :/

**Comment by Acruve15** [comment]
Hey @GHCamille 👋🏼 

I have an additional question. How is this function expected to be used? Will @MateoLostanlen add the pyro-risk repo to the app backend directly? Or something else (using the API for example)

Thx 😃

File: pyro_risks/platform_fwi/get_fwi_effis_score.py (hunk 3 above)
Comment (GHCamille) [reply pyro_risks/platform_fwi/get_fwi_effis_score.py:44]: No it didn't work. I don't know why. I add zero match between point and polygon.

**Comment by GHCamille** [comment]
I'm not sure how @MateoLostanlen wants to use it ! @MateoLostanlen any ideas ?

File: pyro_risks/platform_fwi/get_fwi_effis_score.py (hunk 2 above)
Comment (GHCamille) [reply pyro_risks/platform_fwi/get_fwi_effis_score.py]: done, thanks for the tip !

**Comment by MateoLostanlen** [comment]
Hello @Acruve15 @GHCamille, Hi, there,

Sorry for the late reply, ideally the less you install on the pi the better. Especially since installing a whole repo for one request doesn't seem very adapted. The best thing would be to deploy a small api that the pi can easily call. Is that feasible for you?
Do you need help ?

**Comment by MateoLostanlen** [comment]
smothing like this :

```
import requests

def get_risk_score(lat, lon):
    url = "https://example.com/risk_score"
    params = {
        'lat': lat,
        'lon': lon
    }
    
    try:
        response = requests.get(url, params=params)
        response.raise_for_status()  # Vérifie si la requête a échoué
        data = response.json()  # Supposons que l'API retourne une réponse JSON
        return data['risk_score']  # Supposons que le score de risque se trouve sous la clé 'risk_score'
    except requests.exceptions.RequestException as e:
        print(f"Une erreur est survenue: {e}")
        return None
```

**Pull request merged**